### PR TITLE
Preparation for release v2023.2.1 (Zonemaster/Zonemaster)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

Preparation for release (fix).

## Changes

Updates file `Changes`.

## How to test this PR

Review the changes.

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+ [Fixes]
+ - Fixes bugs in database migration script in Zonemaster-Backend.
+ - Corrects database upgrade instructions (#1244)
+
+ [Zonemaster product]
+ - This version of Zonemaster also consists of the following components. For
+   each component, see its Changes file or Github release notes for complete
+   release information.
+
+   Zonemaster-LDNS      v4.0.0  https://github.com/zonemaster/zonemaster-ldns/releases/tag/v4.0.0
```
Comment (marc-vanderwal) [inline Changes:18]: Shouldn’t that be bumped to 4.0.1?

File: Changes [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+   Zonemaster-Engine    v5.0.0  https://github.com/zonemaster/zonemaster-engine/releases/tag/v5.0.0
+   Zonemaster-CLI       v6.1.0  https://github.com/zonemaster/zonemaster-cli/releases/tag/v6.1.0
+   Zonemaster-Backend  v11.1.0  https://github.com/zonemaster/zonemaster-backend/releases/tag/v11.1.1
```
Comment (marc-vanderwal) [inline Changes:21]: Small oversight here:
```suggestion
   Zonemaster-Backend  v11.1.1  https://github.com/zonemaster/zonemaster-backend/releases/tag/v11.1.1
```

**Review by marc-vanderwal — APPROVED** [review]

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes:18]: Yes, missed that. That's why reviewing is good. :-)

Comment (matsduf) [reply Changes]: Fixed.

**Review by marc-vanderwal — APPROVED** [review]

File: Changes [hunk 3]
```diff
@@ -1,6 +1,25 @@
 Release history of Zonemaster
 
 
+v2023.2.1 2024-03-28
+
+ [Fixes]
+ - Fixes bugs in database migration script in Zonemaster-Backend introduced
+   in release v2023.2.
+ - Corrects database upgrade instructions (#1244)
+
```
[the remark this answers is not in the document]
Comment (matsduf) [reply Changes:10]: No, it is not missing. Here important things are lifted up, and then the details are available in the Changes files of the other components. That is how we have done it before.

File: Changes [hunk 4]
```diff
@@ -1,6 +1,25 @@
 Release history of Zonemaster
 
 
+v2023.2.1 2024-03-28
+
+ [Fixes]
+ - Fixes bugs in database migration script in Zonemaster-Backend introduced
```
[the remark this answers is not in the document]
Comment (matsduf) [reply Changes:7]: Migration failed when using the script before. I think it is fair to call it a bug.

[the remark this answers is not in the document]
Comment (matsduf) [reply Changes]: I have changed the wording to "corrects the script" which is the same wording the Changes file for Zonemaster-Backend.

File: Changes [hunk 5]
```diff
@@ -4,8 +4,8 @@ Release history of Zonemaster
 v2023.2.1 2024-03-28
 
  [Fixes]
- - Fixes bugs in database migration script in Zonemaster-Backend introduced
-   in release v2023.2.
+ - Corrects the database migration script introduced in release v2023.2.in
```
Comment (marc-vanderwal) [inline Changes:7]: Minor detail:
```suggestion
 - Corrects the database migration script introduced in release v2023.2 in
```

Comment (matsduf) [reply Changes]: Every small change risks introducing a new error. :-)

**Pull request merged**